canvas_modules/common-canvas/__tests__/common-properties/controls/structuretable-test.js

```diff
@@ -993,7 +993,7 @@ describe("structuretable columns sort correctly", () => {
 	});
 	it("should sort column alphabetically ascending and descending", () => {
 		// click on the column header to trigger the onClick sort
-		const sortableCol = tableHeader.find("div[role='columnheader']").at(0);
+		const sortableCol = tableHeader.find("div[role='columnheader']").at(1);
```

Comment:
How come this index changed?

Reply:
[I added `columnheader` role to checkbox in the header](https://github.com/elyra-ai/canvas/pull/438/files#diff-bd82dfd3af86850645e442a8dfc5d9ae6ffc7e376a56bf1436f4dd23991733acR126). 
`tableHeader.find("div[role='columnheader']").at(0);` points to the checkbox column in header.
`tableHeader.find("div[role='columnheader']").at(1);` points to column next to checkbox.